ts/components/ButtonToolbar.svelte

```diff
@@ -101,7 +101,13 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
         @include scrollbar.night-mode;
     }
 
-    .wrap-variable {
+    .button-toolbar {
         flex-wrap: var(--buttons-wrap);
+
+        > :global(*) {
+            /* TODO replace with gap once available */
+            margin-inline-end: 0.15rem;
```

Comment:
Just a quick comment for now - please note that there's no guarantee we can move to Qt6 across the platforms yet; it may be the case that a serious issue keeps one or more of the platforms back on 5.14 or .15, so we aren't free to adopt new Chromium features yet

Reply:
That's unfortunate. The `gap` one is the most problematic, because margins won't work if the child has `display: contents`.

Reply:
Oh, I see. I thought the original comment was about the `gap` property, but looking at caniuse, I guess it was about the logical properties. Swapping out with margin-left/bottom is no problem.

Reply:
It was about gap - I presumed you were referring to https://caniuse.com/flexbox-gap

But it looks like margin-inline etc suffer from the same problem :-)